playbooks/roles/edxapp/tasks/main.yml

```
@@ -48,6 +48,15 @@
     - devstack
     - devstack:install
 
+- name: add paver autocomplete to bashrc
+  lineinfile:
+    dest: /root/.bashrc
+    line: "source scripts/paver_autocomplete.sh"
+  when: devstack is defined and devstack
+  tags:
+  - devstack
+  - devstack:install
```

**Reviewer Questions:** 
1. Should I keep `ignore_errors` from original command?
2. I copy/pasted these tags.  I don't actually know whether they are appropriate. :)

---

I think you don't want to ignore errors, and the tags look appropriate.